CarRentals/CarRentals/Program.cs

```diff
@@ -16,14 +16,12 @@ static void Main(string[] args)
             
             CarCRUD option = new CarCRUD();
```

Comment:
maybe we can find a better name for this object? also, remember to use implicit types when possible.

Reply:
Changed it